src/assets/scss/bulma-build.scss

```diff
@@ -1,8 +1,3 @@
-@import "bulma/sass/utilities/functions";
```

Comment:
2 first lines has been removed as the content should be included from "bulma/bulma".